controllers/user.js

```diff
@@ -53,8 +57,10 @@ export const update = async interaction => {
           throw new Error("Error updating user");
         }
         const newScore = doc.score + score;
+        // doc.isCoreTeam =
```

Comment:
I think it would be nice to remove this comment

Reply:
Yes! Thanks for notify.